server/server/settings.py

```diff
@@ -40,6 +40,7 @@
     'rest_framework',
     'github.apps.GithubConfig',
     'projects.apps.ProjectsConfig',
+    'events',
```

Comment:
I'm not entirely sure what you did differently in your configuration of the `events` app, but try to follow the naming conventions of the previous two apps.

Reply:
Fixed